### DISCUSSION TITLE
Add Base Switching Button to Compare Changes

### DISCUSSION THREAD

**Discussion by trajan0x** [community/community#108526] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

I often  find myself needing to swap the base and the target. Usually the workflow goes something like this. In this example I'll be showing a diff between `contracts/comunication` and `master`:

1.  I wanna see a diff so I go Repo Landing Page -> Pull Request
2. I click New Pull Request
3. I put my branch in the first box (this is just me mistake, but I've been using github for a decade and still tend to do this every time)

My screen now looks like this.
 
<img width="998" alt="image" src="https://github.com/community/community/assets/83933037/60380897-02e4-40e9-8649-aed97eb1ba7e">

4. Realizing my mistake, I change the first box branch to master. This causes a page reload that takes a bit (especially if I do it the other way first)
5. I then finally go back to the last box and switch it to `contracts/communication`

One thought I had on fixing this is changing this icon on the left:
<img width="447" alt="image" src="https://github.com/community/community/assets/83933037/4875c372-7e8c-45a3-ac8d-d8f681e95289">

**Comment by oxygen-dioxide** [comment]
I've written a user script that adds a "swap" button to GitHub compare pages. https://github.com/oxygen-dioxide/github-compare-swap/blob/main/github-compare-swap.user.js
